experimental/rust-rocket/README.md

```diff
@@ -0,0 +1,86 @@
+# Rust Rocket Stack
+
+The Rocket stack extends the [Rust stack](../rust/README.md) and provides a consistent way of developing web applications using [Rocket](https://rocket.rs/) applications. Rocket is a fast web framework for Rust that makes it simple to write secure web applications without sacrificing flexibility, usability, or type safety.
+
+This stack is based on the `Rust v1.37` and `Rocket v0.4.2` and allows you to develop new or existing Rocket applications using Appsody.
+
+Currently, debugging is not supported in this stack.
+
+## Health checking
+
+Health-checking enables the cloud platform to determine the `liveness` of your application.
+
+ This exposed as follows:
```

Comment:
nit: This exposed as follows: -> This **is** exposed as follows

Reply:
Done